### ISSUE TITLE
SP950 filter

### ISSUE THREAD

**Issue by RainbowShatter** [LMMS/lmms#7300]
### Enhancement Summary

i think the idea of having a plugin that has a similar sound to what the sp950 filter does, it would make it get a little more of attraction, since i came from ableton live and i used a lot of this plugin for some projects, but is there any for LMMS?

### Implementation Details / Mockup

it should appear in the same place the normal equalizer and low pass filter are, in the effects chain as an option plugin

### Please search the issue tracker for existing feature requests before submitting your own.

- [X] I have searched all existing issues and confirmed that this is not a duplicate.

**Comment by zonkmachine** [comment]
> sp950 filter

Can you please link to this machine/plugin!?

**Comment by zonkmachine** [comment]
@RainbowShatter Can you please add/link to some more info on the SP950?

**Comment by Rossmaxx** [comment]
https://wavetracing.com/products/sp950

This is the sp950 filter in question. Looks like some sort of a 90s drum machine type filter (used in boom bap beats) or for lofi beats. We do have the VyNill ladspa plugin which is similar but is very buggy to start with.

**Comment by Rossmaxx** [comment]
I believe a simple hi pass filter is enough to emulate this. Will close once the OP gets back

**Comment by Rossmaxx** [comment]
Consolidated to #7103

**Comment by Rossmaxx** [comment]
Asked this question to lost on discord and found out that bitcrusher is what @RainbowShatter has been looking for.

**Comment by zonkmachine** [comment]
> Asked this question to lost on discord and found out that bitcrusher is what @RainbowShatter has been looking for.

Well, if it's the dac grit of the 80's samplers we're after it's the A-law/u-law compressors you want. It's in the SWH Lv2 suite.

**Comment by LostRobotMusic** [comment]
The SP950 plugin's central slider similar (or perhaps identical) to the sample rate knob in LMMS's native Bitcrush effect.  The filter knob is a butterworth lowpass (with a 36 dB slope), identical to almost all lowpass filters built into LMMS (except it claims to be "analog", but in practice this is usually a meaningless term unless it has some nonlinearities thrown in).  Some arbitrary filtering is applied before and after that main processing to match it with the hardware it's emulating.

TL;DR @RainbowShatter Set up your signal chain as Equalizer -> Bitcrush -> Equalizer and you'll be able to create sounds nearly identical to everything the plugin's capable of, but with a lot more versatility.

**Comment by RainbowShatter** [comment]
> https://wavetracing.com/products/sp950
> 
> This is the sp950 filter in question. Looks like some sort of a 90s drum machine type filter (used in boom bap beats) or for lofi beats. We do have the VyNill ladspa plugin which is similar but is very buggy to start with.

Oh wow uh thanks that was exactly what i was looking for, great job! :D
